README.md

```diff
@@ -325,11 +325,18 @@ If you need access to the full dataset, you can read the event files that
 TensorBoard consumes by using the [`summary_iterator`](https://github.com/tensorflow/tensorflow/blob/e7f333b5f8b3c53b21d149d8d14c0cebbde431aa/tensorflow/python/summary/summary_iterator.py#L313)
 method.
 
+### Can I customize which lines appear in a plot?
 
-### Can I overlap multiple plots?
+Using the [custom scalars plugin](plugins/custom_scalar), you can create 
```

Comment:
These links don't work for me, they go to this page which is a 404:
https://github.com/chihuahua/tensorboard/blob/37a4c5b2cd91d836d26159fcff3d9365dfd27275/plugins/custom_scalar

I think you want to link to `tensorboard/plugins/custom_scalar` instead, perhaps?

Reply:
+1 Done